cmd/environment_deployments.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/olekukonko/tablewriter"
+	"os"
+	"time"
+
+	"github.com/mglaman/lagoon/graphql"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/spf13/cobra"
+)
+
+var environmentDeploymentsCmd = &cobra.Command{
+	Use:   "deployments",
+	Short: "Environment information",
+	Run: func(cmd *cobra.Command, args []string) {
+		if cmdProject.Name == "" || cmdProject.Environment == "" {
```

Comment:
This doesn't allow you to provide a proper environment name, specifically for PRs